### ISSUE TITLE
We're not ready to surrender yet, even though we're out of ships.

### ISSUE THREAD

**Issue by johndh** [endless-sky/endless-sky#437]
This has happened to me several times, on New Arabia, Huginn Station, and Valhalla.  I enter the system, hail the planet/station, demand tribute, disable/kill/capture the fleet that comes out of the planet/station, disable/kill/capture all of the enemy fleets flying into the system for the next half hour or so, and still the planet/station isn't ready to surrender.  It seems like they should either be fighting back or surrendering, right?  In these cases, I have to leave the system, come back, and try again, and sometimes it works.

**Comment by brendanjones** [comment]
It's the fact that you captured the ships. I'd forgotten about this! No idea if it's a known bug. 

So if a planet defence ship is disabled, it's counted as defeated. But as soon as you capture it, it's 'alive' again, and the planet thinks you haven't finished with their defence fleet. 

The only way I could get around it was to destroy the ship. I've no idea if jumping out then jumping back in wipes that status. Selling the ship certainly does, but I assume that once you jump out of a system you've demanded tribute from, then next time you jump back in wanting to collect that tribute, you'd have to destroy their entire defence fleet again?

**Comment by anonybosch** [comment]
I've noticed that the planet's responses are contingent on your combat rating. When mine was low less than 8, I'd get a "This is a joke, right?" (paraphrasing).
But then as my rating went up from killing wave after wave of pirates, I'd hail the planet again and get first, "We're not ready to surrender yet." then followed by, "We're sending our fleet out." Destroying *that* fleet caused surrender.

**Comment by endless-sky** [comment]
Oops. Commit 1d0bc50 will now count captured defenders as destroyed.

**Comment by brendanjones** [comment]
I'm looking forward to 0.8.9 more and more everyday :D Parking over the planet, demanding tribute, then pillaging all the disabled ships when you're done is quite fun!

**Comment by thenerdfreak** [comment]
I've isolated another case of this. When you take over one planet in a system, the other planets will be unconquerable, saying "We are not ready to surrender yet" even if all ships are destroyed (didn't capture any either)

**Comment by endless-sky** [comment]
I've tested demanding tribute from two planets in one system, and it works fine for me. Since the bug having to do with capturing ships is fixed, I'm closing this, but if you're able to reproduce the bug when trying to take over two planets, feel free to open a new issue.
